index.bs

```diff
@@ -3183,10 +3183,14 @@ associated with or [=scoped=] to, respectively.
                 on {{PublicKeyCredentialEntity/name}}'s value prior to displaying the value to the user or
                 including the value as a parameter of the [=authenticatorMakeCredential=] operation.
 
-          - When inherited by {{PublicKeyCredentialUserEntity}}, it is a [=human palatability|human-palatable=] identifier for a
-            [=user account=]. It is intended only for display, i.e., aiding the user in determining the difference between user
-            accounts with similar {{PublicKeyCredentialUserEntity/displayName}}s. For example, "alexm", "alex.mueller@example.com"
-            or "+14255551234".
+          - When inherited by {{PublicKeyCredentialUserEntity}}, it is a
+            [=human palatability|human-palatable=] identifier for a [=user account=]. This
+            identifier is the primary value displayed to users by [=Clients=] to help users
+            understand with which [=user account=] a credential is associated.
+
+            Examples of suitable values for this identifier include, "alexm", "+14255551234",
+            "alex.mueller@example.com", "alex.mueller@example.com (prod-env)",
+            "alex.mueller@example.com (ACME Corp.)", or "alex.mueller@example.com (ОАО Примертех)".
```

Comment:
I think with the addition of the (prod-env) example, we can drop the (ACME Corp.) one.

```suggestion
            "alex.mueller@example.com", "alex.mueller@example.com (prod-env)",
            or "alex.mueller@example.com (ОАО Примертех)".
```

Reply:
Changes made ✌️